Feature/unr 5043 improve player disconnect reliability (#3001)
spatialos/UnrealGDK: CHANGELOG.md

- Fixed a warning about whitelisted files which was produced in the ExampleProject when building assemblies for cloud deployments.
- Fixed a bug where on initial replication, actors with replicated TArrays would not have the array cleared if the local state was not empty.
- Fixed an issue with replicating references to stably named dynamically added subobjects of dynamic actors.

> - Fixed an issue with player disconnect where controllers, characters, and the client workers were not cleaned up correctly.

Reviewer: ```suggestion
- Fixed an issue during client logout where a client's corresponding Actors were not cleaned up correctly.
```
